docs/src/frontend/index.md

```diff
@@ -760,20 +760,28 @@ On Algorand, it can provide a wallet that directly connects to the Algorand netw
 The network connection is specified via the `providerEnv` key, which may be a string (which is used as an argument to `{!js} providerEnvByName`) or an environment (which is used as an argument to `{!js} setProviderByEnv`).
 By default, signing is via an interactive browser window prompt, where the user repeatedly provides their mnemonic.
 
-If the key `MyAlgoConnect` is provided, and bound to the export of `@reach-sh/stdlib/ALGO_MyAlgoConnect`, then [My Algo](https://wallet.myalgo.com/home) will be used for signing.
+If the key `MyAlgoConnect` is provided, and bound to an export of `@reach-sh/stdlib`, then [My Algo](https://wallet.myalgo.com/home) will be used for signing.
 For example, this sets the wallet fallback to be My Algo used with Algorand TestNet:
 ```js
-import MyAlgoConnect from '@reach-sh/stdlib/ALGO_MyAlgoConnect';
-stdlib.setWalletFallback(stdlib.walletFallback({
-  providerEnv: 'TestNet', MyAlgoConnect }));
+import { loadStdlib, wallets } from '@reach-sh/stdlib';
+const reach = loadStdlib('ALGO');
+const { MyAlgoConnect } = wallets.ALGO;
+reach.setWalletFallback(reach.walletFallback({
+  MyAlgoConnect,
+  providerEnv: 'TestNet',
+}));
 ```
 
-If the key `WalletConnect` is provided, and bound to the export of `@reach-sh/stdlib/ALGO_WalletConnect`, then [WalletConnect](https://walletconnect.com/) is used to connect to the [Algorand Wallet](https://algorandwallet.com/) for signing.
+If the key `WalletConnect` is provided, and bound to the `wallets.ALGO` export of `@reach-sh/stdlib`, then [WalletConnect](https://walletconnect.com/) is used to connect to the [Algorand Wallet](https://algorandwallet.com/) for signing.
```

Comment:
"... and bound to the `wallets.ALGO.WalletConnect` export ..."

Reply:
[bc89dab](https://github.com/reach-sh/reach-lang/pull/664/commits/bc89dabffbb9fbbd23289ff8313f8dc1723415a0) should address this.

Reply:
[e7a6dc8](https://github.com/reach-sh/reach-lang/pull/664/commits/e7a6dc85c8039351f86a3b2ed7dd2312371259c8) should address this.